Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM chabad360/hugo
 
-RUN apk add --update --no-cache openssl git && \
+RUN apk add --update --no-cache openssl git go && \
```

Comment:
This needs to be put in [chabad360/hugo-actions](https://github.com/chabad360/hugo-actions).